OracleWebCenterContent/README.md

```diff
@@ -0,0 +1,16 @@
+# Oracle WebCenter Content Container Image
+
+This project offers scripts to build an Oracle WebCenter Content container image based on 12.2.1.4.0 release. Use this documentation to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
```

Comment:
This is a top-level page, so the link should go to the top-level of the Oracle WebCenter Content docs, not a specific version. Otherwise you'll need to change it often.